package.json

```diff
@@ -18,7 +18,7 @@
     "esdoc": "esdoc -c esdoc.json",
     "esdoc:clean": "rimraf ./esdoc",
     "lint": "npm run lint:js",
-    "lint:eslint": "eslint --ignore-path .gitignore",
+    "lint:eslint": "eslint --ignore-path .gitignore --fix",
```

Comment:
This is related?

Reply:
yeah!
well, debatable whether it is part of the initial PR, but it helped ;-)
It was in its own commit though!